Sources/SentryCrash/Recording/Tools/SentryCrashMachineContext.c

```diff
@@ -172,23 +172,22 @@ isThreadInList(thread_t thread, SentryCrashThread *list, int listCount)
 #endif
 
 void
-sentrycrashmc_suspendEnvironment()
+sentrycrashmc_suspendEnvironment(__unused thread_act_array_t *suspendedThreads,
+    __unused mach_msg_type_number_t *numSuspendedThreads)
```

Comment:
Why are these marked as `__unused` while they clearly are being used?
Possibly because their usage is behind the `SentryCrashCRASH_HAS_THREADS_API` define?

Reply:
Hm, no idea. Do you think we could remove `__unused`?

Reply:
I think its okay to leave in, if I understand correctly, its only there to silence possible "unused variable" warnings. I’m not sure in which circumstances you would *not* define that `HAS_THREADS_API`?

Reply:
Thanks, I removed them.